### PR TITLE
Area view unit list amount

### PR DESCRIPTION
**Pull request by mikkoisa**
- Added overlapping district units to list of units
- Updated wording: services -> units

### CODE REVIEW

File: src/views/AreaView/components/DistrictUnitList/DistrictUnitList.js [hunk 1]
```diff
@@ -87,7 +87,19 @@ const DistrictUnitList = (props) => {
         </div>
       );
     }
-    const districtsWithUnits = district.data.filter(obj => obj.unit);
+    const districtsWithUnits = [];
+    district.data.forEach((obj) => {
+      if (obj.unit) {
+        districtsWithUnits.push(obj);
+      }
+      if (obj.overlaping) {
```
Comment (mhieta) [inline src/views/AreaView/components/DistrictUnitList/DistrictUnitList.js:95]: If this `overlaping` -attrbute is defined by us, then it should be changed to `overlapping`

**Review by mhieta — CHANGES_REQUESTED** [review]
@mikkoisa Please update attribute-typos.

File: src/i18n/en.js [hunk 2]
```diff
@@ -47,8 +47,8 @@ const translations = {
   'area.tab.publicServices': 'Public service areas',
   'area.tab.geographical': 'Neighbourhoods and postal code areas',
   'area.services.local': 'Services in your own area',
-  'area.services.nearby': 'Services in nearby areas as a list',
-  'area.services.all': 'Services as a list',
+  'area.services.nearby': 'Units in nearby areas as a list',
+  'area.services.all': 'Units as a list',
```
Comment (Dremars) [inline src/i18n/en.js:51]: Unit should be Service location(s)

**Review by Dremars — CHANGES_REQUESTED** [review]

File: src/views/AreaView/components/DistrictUnitList/DistrictUnitList.js (hunk 1 above)
Comment (mikkoisa) [reply src/views/AreaView/components/DistrictUnitList/DistrictUnitList.js]: Fixed 👍

File: src/i18n/en.js (hunk 2 above)
Comment (mikkoisa) [reply src/i18n/en.js]: Fixed 👍

**Review by mhieta — APPROVED** [review]
Looks good now 👍

**Review by Dremars — APPROVED** [review]

**Pull request merged**